docs/howtoguides/how_to_simulate_attach.rst

```diff
@@ -31,3 +55,19 @@ automatically enabled when running an ``attach`` operation.
 
     If you want more information about the **AVAILABLE** and **ENTITLED**
     columns, please refer to :ref:`this explanation <pro-status-output>`.
+
+If there are services that will be auto-enabled that you don't want to enable,
+or you only want to enable specific services, you may want to run ``attach``
+:ref:`using a config file <attach-with-config>` instead of the usual attach
```

Comment:
Should we add an example of using the attach config file here?

Reply:
On reflection I'd say probably not here, since it's already covered in the "attach with a config file" page (which has an example config file and the command needed to attach)